server/server.go

```diff
@@ -90,6 +90,7 @@ func New(c *Config) (*Server, error) {
 	cc, err := githubapp.NewDefaultCachingClientCreator(
 		c.Github,
 		githubapp.WithClientUserAgent(userAgent),
+		githubapp.WithClientTimeout(c.Workers.ClientTimeout),
```

Comment:
If the property is not set, passing the zero value is equivalent to no timeout?

Since you can't turn off queued background processing, I think we should set a default timeout that users can increase or decrease, but not disable. Not sure what the value should be though, maybe 10s? 30s?

Reply:
correct:

```
	// A Timeout of zero means no timeout.
```

i like 10s as a start